types/react-redux/react-redux-tests.tsx

```diff
@@ -430,15 +430,6 @@ function mapDispatchToProps(dispatch: Dispatch) {
     };
 }
 
-connect(
-    mapStateToProps,
-    mapDispatchToProps
-)(Counter);
-
-@connect(mapStateToProps)
```

Comment:
Because `connect` no longer returns a class, I guess it can't be used as a decorator.